content/applications/websites/ecommerce/products/additional_features.rst

```diff
@@ -0,0 +1,157 @@
+===================
+Additional features
+===================
+
+Odoo eCommerce offers some additional :ref:`buttons
+<ecommerce/products/additional_features/buttons>` and :ref:`features
+<ecommerce/products/additional_features/product-highlight>` that can be enabled and used throughout
+the entire ecommerce shop, i.e., on the product page, the catalog and/or header.
+
+.. _ecommerce/products/additional_features/buttons:
+
+Buttons
+=======
+
+Several additional buttons can be added to your ecommerce shop, such as the
+:ref:`compare <ecommerce/products/additional_features/product-comparison>` or the
+:ref:`wishlist <ecommerce/products/additional_features/wishlists>` button.
+
+.. image:: additional_features/wishlist-button.png
+   :alt: Additional feature buttons
+
+.. _ecommerce/products/additional_features/product-comparison:
+
+Product comparison
+------------------
+
+The :icon:`fa-exchange` (:guilabel:`Add to compare`) allows website visitors to compare products
+based on their attributes. Customers can click this button for the products they wish to compare,
+then select :guilabel:`Compare` in the pop-up window at the bottom of the page
+to see the comparison summary.
+
+.. image:: additional_features/product-comparison.png
+   :alt: Product comparison window
+
+This button can be enabled independently in different sections of the website:
+
+.. tabs::
+
+   .. group-tab:: Product page
+      Go to a product page, open the :doc:`website editor <../../website/web_design>`, go to the
+      :guilabel:`Style` tab, navigate to the :guilabel:`Products Details` section, and toggle the
+      :icon:`fa-exchange` :guilabel:`Compare` button. Once done, it becomes available on each
+      product page that has attributes.
+
+   .. group-tab:: Catalog page
+
+      To display the :icon:`fa-exchange` (:guilabel:`Compare`) button when hovering over the
+      product on the :ref:`shop page <ecommerce/catalog/product-listing>`, open the website
+      editor, go to the :guilabel:`Style` tab, and click the :icon:`fa-paint-brush` icon next to
```

Comment:
You didn't add a doc link to the website editor here. I don't think it's necessary, as it's very basic and somewhat clutters the page. However, if you add it to the product page tab, I would also add it to the catalog page tab, since users may not have read the product page instructions. However, I would simply remove such links.